pallets/collator-selection/src/lib.rs

```diff
@@ -635,8 +636,18 @@ pub mod pallet {
 
         /// Reset the performance map to the currently active validators at 0 blocks
         pub fn reset_collator_performance() {
-            <BlocksPerCollatorThisSession<T>>::remove_all(None);
```

Comment:
was `remove_all` .... removed?

Reply:
Yeah I tagged all of the changes with the corresponding issue, I guess they only show up in the conversation tab now, as resolved comments.